xbmc/input/KeyboardStat.cpp

```diff
@@ -32,6 +32,8 @@
 #include "threads/SystemClock.h"
 #include "utils/log.h"
 
+#define HOLD_THRESHOLD 250
```

Comment:
with gamepads, a shorter duration can feel more responsive, and a gamer familiar with their controller might want a more responsive device. not sure i can make the same argument for keyboards though